editor/src/components/canvas/canvas-strategies/strategies/reparent-helpers/reparent-strategy-parent-lookup.ts

```diff
@@ -105,6 +105,31 @@ export function getReparentTargetUnified(
   return targetParentUnderPoint
 }
 
+function findConditionalWithEmptyBranch(
```

Comment:
Minor, but maybe the function name should reflect that this is a recursive search.